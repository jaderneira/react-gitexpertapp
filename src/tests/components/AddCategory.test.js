import React from 'react';
import { shallow } from 'enzyme';
import { AddCategory } from "../../components/AddCategory";

describe('Tests in <AddCategory />', () => {

    const setCategories = () => {}
    const wrapper = shallow(<AddCategory setCategories={ setCategories }/>);
        
    test('should show <AddCategory /> correctly', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('should change the text input', () => {
        const input = wrapper.find('input');
        const value = 'Hola mundo'
        
        input.simulate('change', { target: { value } });
        
    })
    
    
})
