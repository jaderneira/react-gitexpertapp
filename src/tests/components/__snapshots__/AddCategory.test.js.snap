// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <AddCategory /> should show <AddCategory /> correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <p>
     
     
  </p>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
